src/main/java/klerer/projectile/ProjectileGraph.java

```diff
@@ -0,0 +1,36 @@
+package klerer.projectile;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ProjectileGraph extends JComponent {
+
+    private Projectile projectile = new Projectile(0, 0);
+
+    @Override
+    protected void paintComponent(Graphics g) {
+        super.paintComponent(g);
+
+        g.translate(0, getHeight());
+        g.drawString("(100, 100)", 100, -100);
+        g.setColor(Color.GREEN);
+        g.drawLine(0, 0, getWidth(), -getHeight());
+        g.drawRect(200, -200, 50, 50);
+        g.setColor(Color.MAGENTA);
+        g.fillRect(400, -400, 25, 25);
+        g.setColor(Color.ORANGE);
+        g.drawOval(200, -200, 50, 50);
+
+        setProjectile(projectile);
```

Comment:
You are setting the projectile to itself.

This method must be called when the values in the frame change.

Reply:
how do I do that?

Reply:
the numbers in my table for like peak Y and intercept X

Reply:
> how do I do that?

When the value changes in the frame, that's when you call setProjectile

Reply:
i dont know how to do that

Reply:
Yes you do. Because you did it for homework last week.

Reply:
do i need a listener?

Reply:
Look at what you did last week.

Reply:
i did but i dont understand how this is the same thing

Reply:
When I say 
"When the value changes in the frame"
What do I mean? What code am I talking about?

Reply:
the values in ProjectileFrame

Reply:
And what code gets executed now when those values change...?

Reply:
the labels are set to the results of the method thats called using those values.
labelX.setText(Double.toString(projectile.getX()));
        labelY.setText(Double.toString(projectile.getY()));
        peakY.setText(Double.toString(projectile.getPeakY()));
        interceptX.setText(Double.toString(projectile.getInterceptX()));
       
      like this part

Reply:
And now you need to set the same Projectile to the Graph

Reply:
do i do that in ProjectileGraph or ProjectileFrame?

Reply:
You're going to have to figure that out based on your understanding of the code you've already written.

Reply:
i understand the code but i dont understand how to do it

Reply:
You're going to have to try something and figure it out.

Reply:
i dont understand how to get the information from projectile frame that was set in the labels to projectile graph

Reply:
You have a Projectile that you generate in generateProjectile()

There is a setProjectile() method in ProjectileGraph.

This part is very clear.